app/controllers/suggestions_controller.rb

```diff
@@ -0,0 +1,91 @@
+get "/suggestions" do
+  puts "*******************"
+  puts "current_user.id:#{current_user.id}"
+  puts "*******************"
+  puts "session user id:#{session[:user_id]}"
+  puts "*******************"
```

Comment:
👍  puts debugging is highly effective and helpful. Just be sure to delete debugging code when the issue is resolved.

Reply:
FYI a good way to debug the session:
```ruby
# this will print the session hash to the server terminal
puts session.inspect
# or you could render the session hash to the browser:
session.inspect # needs to be last line in controller action
``